src/modules/ce/components/client/ClientOpportunitiesTable.tsx

```diff
@@ -0,0 +1,88 @@
+import { Paper, Typography } from '@mui/material';
+import React from 'react';
+import { ColumnDef } from '@/components/elements/table/types';
+import useClientDashboardContext from '@/modules/client/hooks/useClientDashboardContext';
+import GenericTableWithData from '@/modules/dataFetching/components/GenericTableWithData';
+import ProjectTypeChip from '@/modules/hmis/components/ProjectTypeChip';
+import { useFilters } from '@/modules/hmis/filterUtil';
+import { clientBriefName } from '@/modules/hmis/hmisUtil';
+import { ProjectDashboardRoutes } from '@/routes/routes';
+import {
+  ClientCeOpportunitySummaryFieldsFragment,
+  GetClientEligibleOpportunitiesDocument,
+  GetClientEligibleOpportunitiesQuery,
+  GetClientEligibleOpportunitiesQueryVariables,
+} from '@/types/gqlTypes';
+import { generateSafePath } from '@/utils/pathEncoding';
+
+const COLUMNS: ColumnDef<ClientCeOpportunitySummaryFieldsFragment>[] = [
+  {
+    header: 'Project Name',
+    key: 'projectName',
+    render: (opportunity: ClientCeOpportunitySummaryFieldsFragment) =>
+      opportunity.project.projectName,
+  },
+  {
+    header: 'Project Type',
+    key: 'projectType',
+    render: (opportunity: ClientCeOpportunitySummaryFieldsFragment) => (
+      <ProjectTypeChip projectType={opportunity.project.projectType} />
+    ),
```

Comment:
Should we include the Status column? I didn't include it because my understanding is that it would always be "eligible/on waitlist"

Reply:
I think its fine to leave it  out